daemon/algod/api/server/common/handlers.go

```diff
@@ -20,13 +20,15 @@ import (
 	"encoding/json"
 	"net/http"
 
+	"github.com/labstack/echo/v4"
+
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/daemon/algod/api/server/lib"
 	"github.com/algorand/go-algorand/daemon/algod/api/spec/common"
 )
 
 // SwaggerJSON is an httpHandler for route GET /swagger.json
-func SwaggerJSON(ctx lib.ReqContext, w http.ResponseWriter, r *http.Request) {
+func SwaggerJSON(ctx lib.ReqContext, context echo.Context) {
```

Comment:
per your change, you already included the echo.Context within the lib.ReqContext, so why do you need to pass it over again ?

Reply:
I decided against making a copy of the `ReqContext` for each REST call, but forgot to remove the Context from it. Removed `echo.Context` from `ReqContext`.